README.md

```diff
@@ -24,7 +24,13 @@ To open and run the project, open the `code/settings.gradle` file in Android Stu
 
 **Run demo application**
```

Comment:
We may need a new section for setting up the mobile property 

**Data Collection mobile property prerequisites**

This test app needs to be configured with the following edge extensions before it can be used:
- Mobile Core (installed by default)
- [Edge](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension)
- [Edge Identity](https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network)

Reply:
Added a new section above the run demo application section.